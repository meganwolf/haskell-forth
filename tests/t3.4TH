Hello STR .
Flo rida CONCAT2 CR .
Mr. Worldwide 300 5 + STR CONCAT3 CR .
