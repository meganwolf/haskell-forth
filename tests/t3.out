Hello
Florida
Mr.Worldwide305